### DISCUSSION TITLE
Why can't I turn off Google Cast scripts when using <mux-video>?

### DISCUSSION THREAD

**Discussion by digitalfreelanceruk** [muxinc/media-elements#58] [Q&A] [answered]
I'm using mux for a simple website auto-playing video and from what I can see, <mux-video> loads this Google script: https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1

Which in turn loads two more scripts:
https://www.gstatic.com/cast/sdk/libs/sender/1.0/cast_framework.js
https://www.gstatic.com/eureka/clank/130/cast_sender.js

Even with a disableremoteplayback attribute which I believe should block use of casting, and a chromecast="false" attribute (I think ChatGPT gave me that suggestion), I can't stop these scripts from loading.

I don't want Google scripts. I don't want unnecessary scripts.

Can anyone advise how best to stop these scripts please?

**Comment by luwes** [comment]
Thanks for the feedback. I think that's a valid request. 

I'll move this to an issue in the repo that contains the castable-video project.
https://github.com/muxinc/media-elements/tree/main/packages/castable-video

**Reply by digitalfreelanceruk** [reply]
Great, thank you. Do you know off-hand that if I used mux-player I'd be able to stop these scripts from loading?

**Reply by luwes** [reply]
At the moment it's not possible I believe.

**Comment by luwes** [comment, marked as answer]
this is supported in the latest release 🎉 

adding the `disableremoteplayback` attribute or in React adding `disableRemotePlayback={true}` will prevent the loading of the cast scripts.
